style.css

```diff
@@ -0,0 +1,521 @@
+  margin-bottom: 120px;
+  padding: 15px;
+}
+
+.col-2 img {
+  width: 100%;
+}
+
+.col-2 .list-container-1 img {
+  border-radius: 50%;
+  width: 8px;
+  height: 8px;
+  margin: 3px;
+}
+
+.col-2-1 p {
+  width: 73%;
+}
+
+.col-2 .list-container-1 ul li {
+  padding-right: 10px;
+  text-align: center;
+  margin: 10px;
+}
+
+/* Contact */
+.contact {
+  background-image: url(images/contact\ form-\ background-shape-desktop.svg);
+  background-repeat: no-repeat;
+  flex-wrap: wrap;
+  width: 100%;
+  height: 100vh;
+}
+
+textarea {
+  width: 447px;
+  height: 170px;
+  color: #cfd8dc;
+}
+
+input {
+  display: flex;
+  margin: 20px;
+  width: 447px;
+  height: 48px;
+  flex-wrap: wrap;
+}
+
+.container-contact {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  margin-top: 10%;
+  height: 400px;
+}
+
+.container-contact p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  width: 40%;
+  vertical-align: top;
+}
+
+.container-contact h2 {
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  vertical-align: center;
+  color: #fff;
+  margin-top: 15%;
+}
+
+/* About my self */
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.about-col-1 .wrapper img {
+  margin: 10px;
+}
+
+.about-myself p {
+  width: 327px;
+  margin: 10px;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+  vertical-align: top;
+  color: #344563;
+}
+
+.about-myself .about-col-1 ul {
+  padding: 10px;
+  color: #505f79;
+  width: 60%;
+  display: flex;
+  list-style-type: none;
+}
+
+.about-myself .about-col-1 ul li a {
+  color: #505f79;
+  text-decoration: none;
+  font-size: 19px;
+}
+
+.about-myself .about-col-1 ul li {
+  padding: 5px;
+  margin-right: 10px;
+}
+
+.down-arrow {
+  display: flex;
+  justify-content: space-between;
+}
+
+.down-arrow i {
+  color: #505f79;
+  margin-left: 100px;
+  display: flex;
+}
+
+.space {
+  margin-bottom: 30px;
+}
+
+.margin {
+  margin: 1%;
+}
+
+#text-color {
+  color: #396df2;
+  margin: 10px;
+}
+
+.below-button {
+  width: 40%;
+}
+
+.below-button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.alignment {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+/* New about my self */
+.about-col-2 {
+  width: 60%;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.about-myself {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.about-myself h1 {
+  color: #091e42;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: left;
+  vertical-align: center;
+  width: 20%;
+  margin: 10px;
+}
+
+.big-button-2:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+/* Media queris */
+@media only screen and (max-width: 600px) {
+  a {
+    text-decoration: none;
+    color: #091e42;
+  }
```

Comment:
- [ ]  As required to this project the media queries for the Desktop breakpoint should use the value for min-width and the property should be in range of 768px or wider, kindly fix this  :wink: